tests/test_unit_tests_threaded.py

```diff
@@ -0,0 +1,2239 @@
+            for future in concurrent.futures.as_completed(future_to_file):
+                filename = future_to_file[future]
+                try:
+                    error = future.result()
+                    if error:
+                        errors.append(error)
+                except Exception as e:
+                    errors.append(
+                        f"Unexpected error processing {filename}: {
+                            str(e)}")
+
+        # If any errors occurred, fail the test with all error messages
+        if errors:
+            self.fail("\n".join(errors))
+
+
+class TestBuilderWithThreads(unittest.TestCase):
+    def setUp(self):
+        # Use the fixtures_dir fixture to set up paths
+        self.data_dir = os.path.join(os.path.dirname(__file__), "fixtures")
+        with open(os.path.join(self.data_dir, "es256_certs.pem"), "rb") as cert_file:
+            self.certs = cert_file.read()
+        with open(os.path.join(self.data_dir, "es256_private.key"), "rb") as key_file:
+            self.key = key_file.read()
+
+        # Create a local Ps256 signer with certs and a timestamp server
+        self.signer_info = C2paSignerInfo(
+            alg=b"es256",
+            sign_cert=self.certs,
+            private_key=self.key,
+            ta_url=b"http://timestamp.digicert.com"
+        )
+        self.signer = Signer.from_info(self.signer_info)
+
+        self.testPath = os.path.join(self.data_dir, "C.jpg")
+        self.testPath2 = os.path.join(self.data_dir, "A.jpg")
+        self.testPath3 = os.path.join(self.data_dir, "A_thumbnail.jpg")
+        self.testPath4 = os.path.join(self.data_dir, "cloud.jpg")
+
+        # Define manifests
+        self.manifestDefinition = {
+            "claim_generator": "python_test",
+            "claim_generator_info": [{
+                "name": "python_test",
+                "version": "0.0.1",
+            }],
+            "format": "image/jpeg",
+            "title": "Python Test Image",
+            "ingredients": [],
+            "assertions": [
+                {'label': 'stds.schema-org.CreativeWork',
+                    'data': {
+                        '@context': 'http://schema.org/',
+                        '@type': 'CreativeWork',
+                        'author': [
+                            {'@type': 'Person',
+                                'name': 'Tester'
+                             }
+                        ]
+                    },
+                    'kind': 'Json'
+                 }
+            ]
+        }
+
+        self.manifestDefinition_1 = {
+            "claim_generator": "python_test_thread1",
+            "claim_generator_info": [{
+                "name": "python_test_1",
+                "version": "0.0.1",
+            }],
+            "format": "image/jpeg",
+            "title": "Python Test Image 1",
+            "ingredients": [],
+            "assertions": [
+                {'label': 'stds.schema-org.CreativeWork',
```

Comment:
we should move away from CreativeWork as the default test example and also ensure we always add a c2pa.created or c2pa.opened, assertion.